ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerSubscription.scala

```diff
@@ -245,12 +237,12 @@ object ParallelIndexerSubscription {
   )(implicit loggingContext: LoggingContext): Batch[DB_BATCH] => Future[Batch[DB_BATCH]] =
     batch =>
       withEnrichedLoggingContext("updateOffsets" -> batch.offsets) { implicit loggingContext =>
+        val started = System.nanoTime()
         dbDispatcher.executeSql(metrics.daml.parallelIndexer.ingestion) { connection =>
           metrics.daml.parallelIndexer.updates.inc(batch.batchSize.toLong)
           ingestFunction(connection, batch.batch)
-          val nowNanos = System.nanoTime()
           metrics.daml.parallelIndexer.ingestion.duration.update(
-            (nowNanos - batch.averageStartTime) / batch.batchSize,
+            (System.nanoTime() - started) / batch.batchSize,
```

Comment:
@mziolekda @tudor-da do we need this metric at all? Why does it make sense to divide this with the batch size?
Isn't this the one which is feeding the indexer load? AFAIR we just one accumulated execution time there, no? That would be already possible without this metric, like this: `daml_parallel_indexer_ingestion_query_mean{benchmark_id="$benchmark_id"}*irate(daml_parallel_indexer_ingestion_query_count{benchmark_id="$benchmark_id"}[30s])`

Reply:
I found this strange too when I noticed it how it is done, but you confirmed my god feeling :)

Reply:
having thinking further, if we decide to implement it independently, metrics part can be purely done on the level of `BatchingParallelIngestionPipe`... we would not need the information from the `Batch` case class (`batchSize`)

Reply:
Yeah, let's drop the normalization 👍 . But I would still keep track of the batch size (so we can leverage it in more detailed analysis)

Reply:
we could still track the batchSize within the stage when it is being batched

Reply:
Nice to see it is getting simpler :raised_hands: